projects/aws/image-builder/builder/utils.go

```diff
@@ -94,7 +104,24 @@ func execCommand(cmd *exec.Cmd) (string, error) {
 	commandOutputStr := strings.TrimSpace(string(commandOutput))
 
 	if err != nil {
-		return "", fmt.Errorf("failed to run command: %v", err)
+		return commandOutputStr, fmt.Errorf("failed to run command: %v", err)
 	}
 	return commandOutputStr, nil
 }
+
+func getGitCommitFromBundle(repoPath string) (string, error) {
+	log.Println("Getting git commit from bundle")
+	loadBundleManifestCommandSequence := fmt.Sprintf("chmod +x %s/build/lib/eksa_releases.sh && source %s/build/lib/eksa_releases.sh && build::eksa_releases::load_bundle_manifest", repoPath, repoPath)
```

Comment:
This whole thing feels a bit strange. Feels like would be better to reimplement this in go, but I think I get why you did this and I suspect we do this kind of thing already since this image builder is really just a wrapper around our build tooling.

The chmod tho, can't we do that just in git and not have to do this everytime?

@vignesh-goutham please review this

Reply:
Ah yeah I was gonna remove that. I put it in code to test locally because it clones the repo everytime and the one on Github doesn't have it.

TBH I shouldn't even need `chmod` in the first place,  but the `source` command run through Go code kept complaining that the file wasn't present even though it was. When run on the terminal, it works fine as it's a shell builtin.